### PR TITLE
Added option to include layered windows in ImageGrab.grab on Windows

### LINKED ISSUE #2569: PIL.ImageGrab.grab() doesn't "see" all windows
Code
```python
im = PIL.ImageGrab.grab()
im.save('2.png')
```

Here is what actually displayed:
image

Here is the screenshot of PIL:
screenshot_27

Pillow version==4.1.0
This is relevant for different pop ups and menus.

According to @glitchassassin (from pyautogui):

 

> Looks like this is a limitation of Pillow, using SRCCOPY instead of SRCCOPY | CAPTUREBLT in the call to BitBlt.

### PR DESCRIPTION
**Pull request by radarhere**
Resolves #2569

### CODE REVIEW

File: docs/reference/ImageGrab.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ or the clipboard to a PIL image memory.
 
 .. versionadded:: 1.1.3
 
-.. py:function:: PIL.ImageGrab.grab(bbox=None)
+.. py:function:: PIL.ImageGrab.grab(bbox=None, includeLayeredWindows=False)
```
Comment (hugovk) [inline docs/reference/ImageGrab.rst]: Let's name this like `include_layered_windows` on the Python side.

Comment (radarhere) [reply docs/reference/ImageGrab.rst]: Okay, I've updated the commit

**Comment by hugovk** [comment]
Thanks! If any of the recently merged PRs need release notes (like this one?), please could you add them?

**Comment by radarhere** [comment]
Okay, I've created PR #3839.

**Pull request merged**